src/static/js/modules/youtube.js

```diff
@@ -7,8 +7,6 @@
 
```

Comment:
Should this entire file be reworked to be "videoplayer" instead of "youtube" as you've done elsewhere?